smtp/smtpc.c

```diff
@@ -378,6 +378,7 @@ smtpc_task_execute(va_list list)
 int
 smtpc_execute(struct smtpc_request *req, double timeout)
```

Comment:
The commits formatting violates [our guidelines](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message):

> 1. Separate subject from body with a blank line.
> 4. Do not end the subject line with a period.

We also don't send fixup commits for review: nobody is interested in it. They should be squashed into appropriate commits, so a reader will always see a patchset in a clean state.

The commit message should close the issue like `Fixes #xxx` or `Closes #xxx`. See https://github.blog/2013-01-22-closing-issues-via-commit-messages/.